trailingstoploss_tp.py

```diff
@@ -430,6 +477,16 @@ def handle_update_deal(thebot, deal, existing_deal, profit_config):
                     True
                 )
 
+                # Check whether there is an old timeout, and log when the new time out is higher than the old time out
+                old_sl_timeout = deal["stop_loss_timeout_in_seconds"]
+                if old_sl_timeout is not None:
+                    if sl_timeout > old_sl_timeout:
+                        logger.info(
+                            f"StopLoss timeout increased from {old_sl_timeout}s "
+                            f"to {int(sl_timeout)}s",
+                            True
+                        )
+
```

Comment:
Waarom mag de timeout alleen maar groter worden? Zie mijn eerdere comment; ik kan best een timeout van 100, 50, 250, en 0 en vier opeenvolgende configs willen gebruiken. Dus volgens mij zou de check hier moeten zijn dat de oude en nieuwe ongelijk zijn aan elkaar. Overigens kan je overwegen om beide if statements in een and samen te voegen, maakt het iets compacter.

Qua naamgeving heb ik overal 'current' en 'new' gebruikt, zou mooi zijn om in die lijn te blijven.